### PR TITLE
Added additional commands (telnet and http protocols) : DPAD and menus

### PR DESCRIPTION
**Pull request by albaintor**
This is a separate PR for just the additional commands. The other PR will contain only the multi instances part
Related to https://github.com/unfoldedcircle/integration-denonavr/issues/33

### CODE REVIEW

**Comment by zehnm** [comment]
Great! And thank you for also opening a PR in the denonavr library 👍 
I'll check it later today when I'm back home.

**Review by zehnm — CHANGES_REQUESTED** [review]
Tested with AVR-X2700H and the new functions work 👍 

Please fix the cosmetic issues, then I'll merge it.

File: intg-denonavr/avr.py [hunk 1]
```diff
@@ -716,6 +718,113 @@ async def select_sound_mode(self, sound_mode: str | None) -> ucapi.StatusCodes:
             return ucapi.StatusCodes.BAD_REQUEST
         await self._receiver.async_set_sound_mode(sound_mode)
 
+    @async_handle_denonlib_errors
+    async def cursor_up(self) -> ucapi.StatusCodes:
+        """Send previous-track command to AVR."""
```
[the remark this answers is not in the document]
Comment (albaintor) [reply intg-denonavr/avr.py:723]: Done

File: intg-denonavr/avr.py [hunk 2]
```diff
@@ -716,6 +718,113 @@ async def select_sound_mode(self, sound_mode: str | None) -> ucapi.StatusCodes:
...
+            if res is not None and res == "MNMEN ON":
+                await self.setup_close()
+            else:
+                await self.setup_open()
+        else:
+            res = await self._receiver.async_send_telnet_commands("MNMEN?")
+            if res is not None and res == "MNMEN ON":
+                await self.setup_close()
+            else:
+                await self.setup_open()
```
[the remark this answers is not in the document]
Comment (albaintor) [reply intg-denonavr/avr.py]: Indeed, I replaced the telnet command to get the info by the get command in all cases

**Comment by albaintor** [comment]
All modifications applied, please check for merge

**Review by zehnm — CHANGES_REQUESTED** [review]
Without the changed files in .idea it's ok.

File: .idea/integration-denonavr.iml
[the remark this answers is not in the document]
Comment (albaintor) [reply .idea/integration-denonavr.iml:1]: I reverted it but my IDE (pycharm CE)  changed it again .... done

File: .idea/misc.xml
Comment (zehnm) [inline .idea/misc.xml:1]: I'll revert this once merged. Python 3.11 is required to embedd it in the R2 firmware.

**Review by zehnm — APPROVED** [review]
Thanks for the enhancement! I'll create a new release and include it in the next firmware release.

**Pull request merged**